median/plugin.go

```diff
@@ -106,9 +134,16 @@ func (c *chainReaderContract) LatestTransmissionDetails(ctx context.Context) (co
 func (c *chainReaderContract) LatestRoundRequested(ctx context.Context, lookback time.Duration) (configDigest ocrtypes.ConfigDigest, epoch uint32, round uint8, err error) {
 	var resp latestRoundRequested
 
-	err = c.chainReader.GetLatestValue(ctx, c.contract, "LatestRoundReported", map[string]any{"lookback": lookback}, &resp)
+	err = c.chainReader.GetLatestValue(ctx, contractName, "LatestRoundRequested", nil, &resp)
 	if err != nil {
-		return
+		if errors.Is(err, types.ErrNotFound) {
+			// If there's nothing on-chain yet, an implementation will not have emitted an event,
+			// or may not find details of a latest transmission on-chain if it's a function call.
+			// A zeroed out LatestRoundRequested tells later parts of the system that there's no data yet.
+			c.lggr.Warn("LatestRoundRequested not found")
```

Comment:
We could include the source error here for completeness, and it would let us double check the `Is` logic for the current test failure:
```suggestion
			c.lggr.Warn("LatestRoundRequested not found", "err", err)
```